robot/EDA/resources/locators.py

```diff
@@ -82,6 +82,12 @@
         "automatic_household_naming": "//span[text()='Automatic Household Naming:']",
         "adminstrative_account_name_format": "//span[text()='Administrative Account Name Format:']",
         "household_account_name_format": "//span[text()='Household Account Name Format:']",
+        "batch_successful": "(//td/following-sibling::td[text()='Batch Apex']/following-sibling::td[text()='Completed'])[1]",
```

Comment:
I understand since there are multiple locators with the same value you have chosen indexes but is there a way to avoid indexes because they tend to break if the indexes change?

Reply:
This is needed in this case.

Reply:
As in, I intentionally want the very first line that is returned.  This is a list, and I need the very first item in the list.  It's intentional. :-)

Reply:
So you are saying you do not worry about the value. You just require the first line within the list.